nectar/config.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from builtins import object
```

Comment:
This looks like it breaks Python 2 and, as far as I can tell, doesn't add Python 3 compatibility:

```
[vagrant@dev ~]$ python
Python 2.7.12 (default, Sep  2 2016, 14:46:00) 
[GCC 6.1.1 20160621 (Red Hat 6.1.1-3)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from builtins import object
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named builtins
>>> quit()
[vagrant@dev ~]$ python3
Python 3.5.1 (default, Aug  9 2016, 15:35:51) 
[GCC 6.1.1 20160621 (Red Hat 6.1.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from builtins import object as builtin_object
>>> object is builtin_object
True
>>> 
```

Can you explain what the goal of this particular import is?